avivator/src/utils.js

```diff
@@ -27,10 +27,14 @@ export async function createLoader(
       // built in to the format for them, hence the ternary.
       const {
         omexml: { SizeZ, SizeT, SizeC },
-        isBioFormats6Pyramid
+        isBioFormats6Pyramid,
+        isPyramid
       } = loader;
       const totalImageCount =
-        SizeZ * SizeT * SizeC * (isBioFormats6Pyramid ? 1 : loader.numLevels);
+        SizeZ *
+        SizeT *
+        SizeC *
+        (isPyramid ? (isBioFormats6Pyramid ? 1 : loader.numLevels) : 1); // eslint-disable-line no-nested-ternary
```

Comment:
This is a different bug I noticed while testing this out.  The previous code was not rigorous enough for non-pyramidal images,

Reply:
Ok